Frends.Community.Azure.Blob.Tests/DeleteTest.cs

```diff
@@ -9,72 +9,122 @@ namespace Frends.Community.Azure.Blob.Tests
     public class DeleteTest
     {
         /// <summary>
-        ///     Connection string for Azure Storage Emulator
+        ///     Connection string for Azure Storage Account.
         /// </summary>
         private readonly string _connectionString = Environment.GetEnvironmentVariable("HiQ_AzureBlobStorage_ConnString");
-
-        /// <summary>
-        ///     Container name for tests
-        /// </summary>
-        private readonly string _containerName = "test-container";
-
-        [TestCleanup]
-        public async Task Cleanup()
-        {
-            // delete whole container after running tests
-            var container = Utils.GetBlobContainer(_connectionString, _containerName);
-            await container.DeleteIfExistsAsync();
-        }
+        private readonly string _appID = Environment.GetEnvironmentVariable("HiQ_AzureBlobStorage_AppID");
+        private readonly string _tenantID = Environment.GetEnvironmentVariable("HiQ_AzureBlobStorage_TenantID");
+        private readonly string _clientSecret = Environment.GetEnvironmentVariable("HiQ_AzureBlobStorage_ClientSecret");
+        private readonly string _storageAccount = "testsorage01";
 
         [TestMethod]
         public async Task DeleteBlobAsync_ShouldReturnTrueWithNonexistingBlob()
         {
+            var containerName = "test1";
+
             var input = new DeleteBlobProperties
             {
                 BlobName = Guid.NewGuid().ToString()
             };
             var connection = new BlobConnectionProperties
             {
+                ConnectionMethod = ConnectionMethod.ConnectionString,
                 ConnectionString = _connectionString,
-                ContainerName = _containerName
+                ContainerName = containerName
             };
-            var container = Utils.GetBlobContainer(_connectionString, _containerName);
+            var container = Utils.GetBlobContainer(_connectionString, containerName);
             await container.CreateIfNotExistsAsync();
 
             var result = await DeleteTask.DeleteBlobAsync(input, connection, new CancellationToken());
 
             Assert.IsTrue(result.Success, "DeleteBlob should've returned true when trying to delete non existing blob");
+            await DeleteContainer(containerName);
         }
 
         [TestMethod]
         public async Task DeleteBlobAsync_ShouldReturnTrueWithNonexistingContainer()
         {
+            var containerName = "test2";
+
             var input = new DeleteBlobProperties
             {
                 BlobName = Guid.NewGuid().ToString()
             };
             var options = new BlobConnectionProperties
             {
+                ConnectionMethod = ConnectionMethod.ConnectionString,
                 ConnectionString = _connectionString,
-                ContainerName = Guid.NewGuid().ToString()
+                ContainerName = containerName
```

Comment:
Why is this changed to be some predefined value instead of a random Guid? Isn't the purpose of the test to check that non-existent container case is ok? What if "test2" container somehow stays alive after some tests?